src/main/java/com/meetme/plugins/jira/gerrit/webpanel/ShowReviewsWebPanelCondition.java

```diff
@@ -66,10 +66,7 @@ public boolean shouldDisplay(Map<String, Object> map) {
 
     private boolean isGerritProject(final Issue issue) {
 
-        if (issue.getProjectId() == null)
-           return false;
+        return issue.getProjectId() != null && !isEmpty(gerritConfiguration.getIdsOfKnownGerritProjects()) && gerritConfiguration.getIdsOfKnownGerritProjects().contains(issue.getProjectId().toString());
```

Comment:
Would like to see this kept to an accessible column limit. Your change puts it all on one line, and that line exceeds 200 chars.